src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -49,6 +47,7 @@
 import java.util.stream.Collectors;
 
 @RunWith(VertxUnitRunner.class)
+@SuppressWarnings("checkstyle:JavaNCSS")
```

Comment:
I think you should be able to annotate the method, rather than the whole class.

Reply:
Reading about the metric I think that in this case it's about the whole class which is becoming quite big even if QE people are working on having it better.